farmdata2_modules/fd2_tabs/fd2_example/cacheSpike.html

```diff
@@ -0,0 +1,35 @@
+<body>
+    <div id='cacheSpike'>
+        <h1>sup</h1>
+        <input data-cy='text-input' type=text v-model='newCache'>
+        <button>Save To Cache</button>
+        <button @click='cacheArray'>Put Array in Cache</button>
+    </div>
+    <script>
+        new Vue({
+            el: '#cacheSpike',
+            data:{
+                newCache: null,
+                testArray: [1, 2, 3, 4],
+                newArray: [],
+            },
+            methods:{
+                cacheArray(){
+                    localStorage.setItem('myArray', JSON.stringify(this.testArray))
+                },
+            },
+            computed:{
```

Comment:
In general, computed properties should not perform any operations that have side effects (change a value or in this case store a value in local storage).  Computed properties should be reserved to generating a new value from others that exist in the data already.